app/api/helpers.py

```diff
@@ -136,54 +151,57 @@ def generate_random_token(length=32):
 def user_info(user):
     from app.api.services import agency_service
     try:
-        user_type = current_user.role
+        user_type = user.role
```

Comment:
This is a bug I noticed - the `user_info` function accepts a `user` instance, but never actually used it, instead it referenced the `current_user` instance.